bubblewrap.c

```diff
@@ -1624,9 +1639,30 @@ main (int    argc,
       exit (0); /* Should not be reached, but better safe... */
     }
 
+  /* Child, in sandbox, privileged in the parent *or* in the user namespace.
+   *
+   * NOTE: This is always ptrace:able in the case of user namespaces
+   * (due to all parent namespaces having CAP_SYS_PTRACE in the child
+   * namespaces), but it should be ok as it has no permissions in the
+   * parent namespace.
```

Comment:
In general, bubblewrap in the non-privileged case is just that...not privileged, any security vulnerabilities are in the kernel.